lib/puppet/file_serving/configuration.rb

```diff
@@ -82,6 +83,8 @@ def mk_default_mounts
     @mounts["plugins"].allow('*') if @mounts["plugins"].empty?
     @mounts["pluginfacts"] ||= Mount::PluginFacts.new("pluginfacts")
     @mounts["pluginfacts"].allow('*') if @mounts["pluginfacts"].empty?
+    @mounts["tasks"] ||= Mount::Tasks.new("tasks")
+    @mounts["tasks"].allow('*') if @mounts["tasks"].empty?
```

Comment:
Generally it looks like we protect everything via the `puppet/v3/file_content` route.